README.md

```diff
@@ -45,43 +47,49 @@ vim.keymap.set('n', '<leader>sp', '<cmd>lua require("spectre").open_file_search(
     desc = "Search on current file"
 })
 ```
-use command: `Spectre`
 
-**WARNING**
-* Commit your file before you replace text. It does not support undo
-* Don't use your crazy vim skill to edit result text or UI.
-* You can use `dd` to toggle result item
-* You need to use `<esc>` not `<c-c>` to leave insert mode.
+Use command: `:Spectre`
 
+## Warnings
 
-## Regex Issue
-* default regex use vim regex **magic mode** `\v` and **no-ignore-case** .
-* It has different regex of search command `rg` and replace command
-`sed` so careful if you want to replace text.
-* It has a different of highlight result because I use vim regex to
-highlight text so careful but you can try to replace.
+- Always commit your files before you replace text. `nvim-spectre`
+  does not support undo directly (Although `nvim` might if you have
```

Comment:
nvim support undo file but it will not work on our plugin.  because sed can replace outside of nvim. if user not open that file it can't restore

Reply:
Ah I see, I've removed that bit now.